CHANGELOG.md

```diff
@@ -6,6 +6,8 @@
 
 ### Feature
 
+- Updated "Volto in Production" list @alecghica
```

Comment:
@alecghica since this has nothing to do with Volto software I wouldn't add a changelog entry here.

Reply:
@tisto check my initial commit. I added the changelog because "Changelog verifier" failed. Shall I just remove the changelog entry and ignore that?

Reply:
Since when do we have a changelog verifier on the Volto repo? If we need a changelog entry this should be under "internal" then.

Reply:
Done.